Subtitle Composer fails to build

## Describe the bug

Subtitle Composer fails to build when added to system packages.

## Steps To Reproduce

Steps to reproduce the behavior:

1. Add Subtitle Composer to system packages
2. Rebuild system
3. Build fails

## Expected behavior

Subtitle Composer succeeds building.

## Additional context

### Error message
```
error: builder for '/nix/store/fa04hif0zd5pc704y23fmnpkvn81a3d3-subtitlecomposer-0.8.1.drv' failed with exit code 2;
       last 25 log lines:
       >       |                                swr_alloc_set_opts2
       > /build/source/src/streamprocessor/streamprocessor.cpp: In member function 'void SubtitleComposer::StreamProcessor::processAudio()':
       > /build/source/src/streamprocessor/streamprocessor.cpp:366:33: error: 'AVFrame' {aka 'struct AVFrame'} has no member named 'channel_layout'; did you mean 'ch_layout'?
       >   366 |                 frameResampled->channel_layout = m_audioChannelLayout;
       >       |                                 ^~~~~~~~~~~~~~
       >       |                                 ch_layout
       > /build/source/src/streamprocessor/streamprocessor.cpp:434:59: error: 'AVFrame' {aka 'struct AVFrame'} has no member named 'pkt_duration'; did you mean 'duration'?
       >   434 |                                                 if(frame->pkt_duration)
       >       |                                                           ^~~~~~~~~~~~
       >       |                                                           duration
       > /build/source/src/streamprocessor/streamprocessor.cpp:435:84: error: 'AVFrame' {aka 'struct AVFrame'} has no member named 'pkt_duration'; did you mean 'duration'?
       >   435 |                                                         timeFrameDuration = frame->pkt_duration * 1000 * m_avStream->time_base.num / m_avStream->time_base.den;
       >       |                                                                                    ^~~~~~~~~~~~
       >       |                                                                                    duration
       > /build/source/src/streamprocessor/streamprocessor.cpp:451:133: error: 'AVFrame' {aka 'struct AVFrame'} has no member named 'channels'
       >   451 |                                                 emit audioDataAvailable(frameResampled->data[0], qint32(frameSize * frameResampled->channels),
       >       |                                                                                                                                     ^~~~~~~~
       > /build/source/src/streamprocessor/streamprocessor.cpp:456:115: error: 'AVFrame' {aka 'struct AVFrame'} has no member named 'channels'
       >   456 |                                                 emit audioDataAvailable(frame->data[0], qint32(frameSize * frame->channels),
       >       |                                                                                                                   ^~~~~~~~
       > [ 80%] Building CXX object src/CMakeFiles/subtitlecomposer-lib.dir/videoplayer/backend/glrenderer.cpp.o
       > make[2]: *** [src/CMakeFiles/subtitlecomposer-lib.dir/build.make:1489: src/CMakeFiles/subtitlecomposer-lib.dir/streamprocessor/streamprocessor.cpp.o] Error 1
       > make[2]: *** Waiting for unfinished jobs....
       > make[1]: *** [CMakeFiles/Makefile2:397: src/CMakeFiles/subtitlecomposer-lib.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
```

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.60, NixOS, 24.11 (Vicuna), 24.11.20241109.76612b1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - channels(root): `"nixos-22.11"`
 - channels(ecm): `""`
 - nixpkgs: `/nix/store/0sxsfzswhjck7f4sbsvznz285s4i7y64-source`

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@kugland 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/su/subtitlecomposer/package.nix
+++ b/pkgs/by-name/su/subtitlecomposer/package.nix
@@ -3,7 +3,7 @@
   fetchFromGitLab,
   cmake,
   extra-cmake-modules,
-  ffmpeg,
+  ffmpeg_6,
   openal,
   stdenv,
   libsForQt5,
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
   ];
   buildInputs =
     [
-      ffmpeg
+      ffmpeg_6
       openal
     ]
     ++ (with libsForQt5; [

